pkg/skaffold/runner/runner.go

```diff
@@ -331,6 +327,15 @@ func (r *SkaffoldRunner) BuildAndTest(ctx context.Context, out io.Writer, artifa
 
 // Deploy deploys the given artifacts
 func (r *SkaffoldRunner) Deploy(ctx context.Context, out io.Writer, artifacts []build.Artifact) error {
+	if len(r.runCtx.Opts.PreBuiltImages) > 0 {
+		logrus.Debugln("Deploy using pre-built images")
+		pBuilts, err := convertImagesToArtifact(r.runCtx.Opts.PreBuiltImages)
+		if err != nil {
+			return errors.Wrap(err, "deploy failed")
+		}
+		// Merge current build artifacts with previously built
```

Comment:
I don't understand this comment. This portion of code is usually called in a dev loop to merge the artifacts built in the last loop and the artifacts built in the previous loops.

Reply:
this is to merge the build artifacts, which is empty now. Something #2001, will add in.

Reply:
What do you think about removing this until #2001 introduces it then?

Reply:
yes. Like i mentioned in the channel, i am going to remove the last commit from this branch. 
I pushed it for David to see the whole change. 
Please re-view this.